mlir/lib/Conversion/SCFToOpenMP/SCFToOpenMP.cpp

```diff
@@ -340,8 +340,10 @@ namespace {
 
 struct ParallelOpLowering : public OpRewritePattern<scf::ParallelOp> {
 
-  ParallelOpLowering(MLIRContext *context)
-      : OpRewritePattern<scf::ParallelOp>(context) {}
+  unsigned numThreads;
+
+  ParallelOpLowering(MLIRContext *context, unsigned numThreads)
```

Comment:
Nit: can we have a default value for `numThreads` here?

Reply:
Done, thanks. I didn't set it here in the first place because the default value it's already set in the .td file.

Reply:
I would be careful about default values in this context. Perhaps rely on a default like "0" to denote "default". This is because OpenMP's default behavior for the default number of threads is "all available threads". Hence for instance defaulting to one ("1") may be counterintuitive wrt OpenMP (although it might actually improve performance ;-).